app/helpers/mb_form_builder.rb

```diff
@@ -0,0 +1,71 @@
+class MbFormBuilder < ActionView::Helpers::FormBuilder
```

Comment:
Naming: Prefixing classes with `Mb` seems reasonable, although the acronym might not be clear to the next developer. Also, isn't everything in the app is for MichiganBridges? I'm probably missing something but just trying to understand the scheme.